bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -388,6 +389,17 @@ def __call__(self, *args):
 
         return res
 
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
+    """
+    Filters the RDF according to the function or string provided.
+    """
+    if isinstance(func, str): # If string argument is passed. Invoke the Orginal Filters.
+        return rdf._OriginalFilter(func)
+    jitter = FunctionSignature(rdf)
+    func.__annotations__['return'] = 'bool' # return type for Filters is bool # Note: You can keep double and Filter still works.
+    func_call = jitter.jit_function(func, funcargs) # JIt the function.
```

Comment:
Shouldn't funcargs be either a list with the column names or the name of the filter (according to the possible signatures of the overloads that accept a callable as first argument)? Why is it a dictionary?

Reply:
Yes. I initially went with an *args design. But it later proved to be an issue with respect to some specific cases.
It would require the user to use *args to mention where each argument of the function is mapped to in that specific order. 
The issue being if the user wanted to pass constants to a function and wanted to remap only few of the columns. It makes it hard to know which of the args should be mapped where. Only exacerbated if the function has default arguments.